woof/code/target-shield-code.html

```diff
@@ -0,0 +1,326 @@
+  })
+  
+  // Create shield
+  var shieald = new Rectangle({
+    x: 50,
+    y: 5,
+    length: 10,
+    height: 120
+  })
+  shieald.hide()
+  
+  var shieald2 = new Rectangle({
+    x: -50,
+    y: 5,
+    length: 10,
+    height: 115
+  })
+  shieald2.hide()
+  
+  // Create text variable to display shield instructions
+  
+  var shiealdText = new Text({
+    text: "Press 'S' for shield - 1 time use",
+    x: 137,
+    y: 230,
+    color: "green"
+  })
+  
+  // Show shield if S is pressed. Delete after 4 seconds.
+  forever(() => {
+    if (keysDown.includes('S')) {
+      shieald.show()
+      shieald2.show()
+      after(4, 'seconds', () => {
+        shieald.delete()
+        shieald2.delete()
+      })
+    }
+  })
+  
+  // Create blocker sprite
+  var bloacker = new Rectangle({
+    height: 37,
+    width: 37,
+    color: "orange"
+  })
+  
+  // Make blocker sprite follow mouse
+  forever(() => {
+    bloacker.x = mouseX
+    bloacker.y = mouseY
+  })
+  
+  // Create timer variable
+  var timer = 0
+  
+  // Increase timer by 1 every second
+  every(1, 'second', () => {
+    timer += 1
+  })
+  
+  // Display timer on screen
+  new Text({
+    text: () => "Time: " + timer,
+    x: 0,
+    y: 230,
+  })
+  
+  // If timer greater than 100, reset health to 100
+  var levealreset = 0
+  forever(() => {
+    if (timer == 100) {
+      levealreset = 1
+    }
+      if (timer == 200) {
+      levealreset = 1
+    }
+    if (levealreset == 1) {
+      haelth = 100
+      after(0.1, 'second', () => {
+        levealreset = 0
+  
+      })
+    }
+  })
+  
+  // Create health variable, start at 100, and display on screen
+  var haelth = 100
+  new Text({
+    text: () => "health: " + haelth,
+    x: 0,
+    y: 200,
+    size: 20
+  })
+  
+  
+  // Create empty array to push clones into
+  var clones = []
+  
+  // Create red circle clone
+  every(1, "seconds", () => {
+    var clone = new Circle({
+        radius: 11,
+        color: "red",
+        x: 0,
+        y: 0
+      })
+      // Movement & Damage
+    baellspawn(clone) // Call baellspawn function
+    clones.push(clone) // Push clones into clones array
+    forever(() => {
+      clone.move(1) // Set initial speed to 1
+        if (timer > 100) {
+          clone.move(1.6) // Increase speed if timer greater than 100
+        }
+        if (timer > 200) {
+          clone.move(1.9) // Increase speed again if timer greater than 200
+        }
+      if (clone.touching(baise)) {
+        haelth -= 5 // Decrease health by 5 if clones touches base
+        clone.delete() // Delete clone
+      }
+    })
+  })
+  
+  // Create green circle clone
+  every(1, 'seconds', () => {
+    var clone2 = new Circle({
+      radius: 7,
+      color: "green",
+      x: 0,
+      y: 0
+    })
+    baellspawn(clone2) // Call baellspawn function
+    clones.push(clone2) // Push clone into clones array
+      //Movement & Damage
+    forever(() => {
+      clone2.move(2.7) // Set initial speed to 2.7
```

Comment:
This would be cleaner
```javascript
forever(() =>
  if (time < 100) {
    clone2.move(2.7)
  }
  else if (timer.between(100, 200)) {
    clone2.move(5.7) // Increase speed if timer greater than 100
  }
  if (timer > 200) {
    clone2.move(9) // Increase speed again if timer greater than 200
  }
})
```